packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -290,14 +269,18 @@ class _DropdownRoute<T> extends PopupRoute<_DropdownRouteResult<T>> {
     assert(style != null);
   }
 
-  final ScrollController scrollController = new ScrollController();
   final List<DropdownMenuItem<T>> items;
   final Rect buttonRect;
   final int selectedIndex;
   final int elevation;
   final ThemeData theme;
   final TextStyle style;
 
+  // These fields are used by _DropdownMenuRouteLayout and initialized by buildPage().
```

Comment:
Good point. Although it seems to DTRT, _DropdownMenuRouteLayout.shouldRelayout() really should be comparing all of the values it depends on.